invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html

```diff
@@ -1,74 +1,76 @@
 {#
   Copyright (C) 2020 TUGRAZ
-  Copyright (C) 2020 mojib wali.
+  Copyright (C) 2020 mojib wali
+  Copyright (C) 2020 Nikita Lvov
 
   invenio-theme-tugraz is free software; you can redistribute it and/or
   modify it under the terms of the MIT License; see LICENSE file for more
   details.
 #}
-
-{%- extends "invenio_theme/page_cover.html" %}
+{%- extends config.ACCOUNTS_COVER_TEMPLATE %}	
 {% from "invenio_accounts/_macros.html" import render_field, form_errors %}
 
+
 {% block page_body %}
-  <div class="ui padded segments big form">
-    <div class="ui segment padded">
-      <div class="divider hidden"></div>
+<div class="ui centered stackable grid">
+  <div class="centered row">
+  <h2>{{ _('Welcome Back!') }}</h2>
+  </div>
+  <div class="ui divider"></div>
+  <div class="ui bottom attached segment sign-form">
+  <div class="ui one column stackable grid">
+    <div class="column">
       {%- block form_header %}
-        <h3 class="ui header">{{ _('Log in to account') }}</h3>
+        <h3 class="ui inverted header">{{ _('Log in to account') }}</h3>
       {%- endblock form_header %}
 
       {%- if config.INVENIO_CONFIG_TUGRAZ_SHIBBOLETH %}
-      <a href="{{ url_for('sso_saml.sso', idp='idp') }}">
-      <div class="ui fluid animated fade large button basic" tabindex="0">
-        <div class="visible content">
-          
-            {% trans type='TUG' %} Log in with {{ type }}{% endtrans %}
-         
-        </div>
-        <div class="hidden content">
-          <img src="{{ url_for('static', filename=config.INVENIO_THEME_TUGRAZ_ICON)}}" height="20px">
-        </div>
-      </div> 
-    </a>
-      
-      <div class="ui horizontal divider">
-        Or
-      </div>
+      <button class="ui icon inverted button" href="{{ url_for('sso_saml.sso', idp='idp') }}">
```

Comment:
```suggestion
      <a href="{{ url_for('sso_saml.sso', idp='idp') }}">
      <button class="ui icon inverted button" >
      </a/
```
The hyperlink is not working, while the ```href``` is inside a button.